src/Parser/grammar.pp

```diff
@@ -0,0 +1,90 @@
+%skip   space               [\x20\x09\x0a\x0d]+
+%skip   doc_                [/**]
+%skip   _doc                [*/]
+%skip   star                [*]
+
+%token  at                  @                           -> annot
+%token  text                [^@].*
+
+%token  annot:identifier    [\\a-zA-Z_][\\a-zA-Z0-9_]*  -> values
```

Comment:
Your suggested change is wrong because you're forcing to have 2 chars before ns separator